hs_core/models.py

```diff
@@ -99,6 +99,22 @@ def clean_for_xml(s):
     return output
 
 
+def get_relation_version_res_url(resource, rel_type):
```

Comment:
I think we should make this an instance method of the `BaseResource` class and check the parameter  `rel_type` value is one of the 2 allowed types.